Makefile

```diff
@@ -14,6 +14,7 @@ build:
 .PHONY: docs
 docs:
 	go mod download
+	echo Using tfplugindocs that needs to be manually installed from https://github.com/hashicorp/terraform-plugin-docs/releases
```

Comment:
I wanted to have this install automatically - let me come back to it.

Reply:
Yes. I have tried to do it but.. 

it will need to be upacked inside the repository. 
It will need to have way to detect operating system using non standard labels they put.

This looks like contribution on their side to add proper installation script ;D

Reply:
I've fixed it in #6 - I needed to use `go install` not `go mod download`